### ISSUE TITLE
Provide a file format system for mutes and other data that is stored as sql

### ISSUE THREAD

**Issue by kinggoesgaming** [hsyyid/EssentialCmds#19]
if a server doesn't use SQL services, then they cant use these services.. which isnt ideal.
This can be a low priority issue as modulating is more important.

Labels: enhancement, low priority

**Comment by hsyyid** [comment]
This is actually the only data stored via SQL, and I agree with you that mutes and other data should probably be reliant on configs, however I feel as though there should be various DATA configs for this sort of thing. For example, we could have the structure similar to the following:

```
+- config
    +- essentialcmds
        +- data
            +- homes.conf
            +- mutes.conf
            +- players.conf
            ...
```

This way, the actually configuration options are separated from the configuration nodes meant to store data.

**Comment by kinggoesgaming** [comment]
Lets wait on other parts of the plugin before we talk on this... I just wanted to have an issue on it when it came to my mind so dont forget about it
